packages/core/package.json

```diff
@@ -16,12 +16,13 @@
     "node": ">=8"
   },
   "dependencies": {
-    "@stoplight/graphite": "4.6.x",
+    "@stoplight/graphite": "5.0.0",
     "axios": "0.x.x",
     "lodash": "4.x.x",
+    "mobx": "^5.9.4",
     "tslib": "1.9.x"
   },
   "devDependencies": {
-    "@types/lodash": "4.x.x"
+    "@types/lodash": "4.14.123"
```

Comment:
Do we want to be pugging to specific versions like this? I guess dependabot will shout at us anyway, but if we did `    "@types/lodash": "^4.14.123"` with the little ^ up there then it says anything above 4.14.123 and below 4.15. Is that better?

Reply:
@philsturgeon the versioning "policy" is that we should use the `1.2.x` pattern (but avoid the `^`). Not sure why @XVincentX didn't use `x`. Probably an oversight.

Reply:
I just let `yarn` set the thing in the way it wants, but if I recall correctly yesterday @P0lip was doing a similar change to have a smaller `node_modules` directory. I can change it if required, I do not mind.

Reply:
Dev dependecies are totally irrelevant since they are trimmed out during the build process, so it can stay as is.

And yeah, I wanted to make the versioning stricter.
We usually use either `^` or `1.x.x` for regular dependencies, but in my case I wanted to make it less relaxed.

Reply:
Done in 32fb515